docs/masking.rst

```diff
@@ -148,9 +157,11 @@ be converted into a CASA image using :func:`~spectral_cube.io.make_casa_mask`:
   >>> from spectral_cube.io.casa_masks import make_casa_mask
   >>> make_casa_mask(cube, 'casa_mask.image', add_stokes=False)  # doctest: +SKIP
 
-Optionally, a redundant Stokes axis can be added to match the original CASA image.
-.. Masks may also be appended to an existing CASA image:
-..  >>> make_casa_mask(cube, 'casa_mask.image', append_to_img=True, img='casa.image')
+Optionally, a redundant Stokes axis can be added to match the original CASA
+image. Masks may also be appended to an existing CASA image::
+
+  >>> make_casa_mask(cube, 'casa_mask.image', append_to_img=True,
```

Comment:
These lines were meant to be commented out, but the comments were improperly formatted.  At the moment, I'm not sure if we ever finished implementing the `append_to_img` functionality... we may have, but if not, we should actually comment this bit out.